python/pyspark/sql/tests.py

```diff
@@ -3551,12 +3544,12 @@ def test_null_conversion(self):
         self.assertTrue(all([c == 1 for c in null_counts]))
 
     def _toPandas_arrow_toggle(self, df):
-        self.spark.conf.set("spark.sql.execution.arrow.enabled", "false")
-        try:
+        with self.sql_conf({"spark.sql.execution.arrow.enabled": False}):
             pdf = df.toPandas()
-        finally:
-            self.spark.conf.set("spark.sql.execution.arrow.enabled", "true")
-        pdf_arrow = df.toPandas()
+
+        with self.sql_conf({"spark.sql.execution.arrow.enabled": True}):
```

Comment:
We can omit this when we use the default value or set the value in setup method, but I'm okay if we want to show the value explicitly.

Reply:
Ah, OK. I am fine. will omit this.